src/utils_format_kairosdb.h

```diff
@@ -41,7 +41,8 @@ int format_kairosdb_value_list(char *buffer, size_t *ret_buffer_fill,
                                size_t *ret_buffer_free, const data_set_t *ds,
                                const value_list_t *vl, int store_rates,
                                char const *const *http_attrs,
-                               size_t http_attrs_num, int data_ttl);
+                               size_t http_attrs_num, int data_ttl,
+                               char *metrics_prefix);
```

Comment:
This is the missing `const` @rpv-tomsk was talking about. This should be `char const *metrics_prefix`.

Reply:
ok thanks, I'm on it, I'll amend my previous commit.